app/assets/javascripts/constants/image_constants.es6.jsx.erb

```diff
@@ -6,6 +6,15 @@
         orange: "<%= asset_path("logo/logo_orange.png") %>"
       };
     }
+
+    get headshots() {
```

Comment:
Yeah you would although I'd think most react components wouldn't need to show images. If you are using something like a sprite, you can put it in the scss instead as a background image. You get rails helpers there